rero_ils/modules/notifications/jsonschemas/notifications/notification-v0.0.1.json

```diff
@@ -52,6 +52,33 @@
       "type": "integer",
       "title": "Current reminder count"
     },
+    "status": {
+      "title": "Status",
+      "type": "string",
+      "default": "created",
+      "enum": [
+        "created",
+        "canceled",
+        "done"
+      ],
+      "form": {
+        "type": "selectWithSort",
+        "options": [
+          {
+            "label": "created",
+            "value": "created"
+          },
+          {
+            "label": "canceled",
+            "value": "canceled"
+          },
+          {
+            "label": "done",
+            "value": "done"
+          }
+        ]
+      }
+    },
```

Comment:
There are no need to declare `form` block because, notification editor doesn't exists. This will prevent translation works